table/sstable/table.go

```diff
@@ -85,6 +108,10 @@ type Table struct {
 
 	oldBlockLen int64
 	oldBlock    []byte
+
+	// For plr
+	plrFd *os.File
+	plr *plrSegments
```

Comment:
`plrFd` is probably not needed?

I guess in the `loadIndexData` below you could do something like 
```golang
        out, err := exec.Command("./plr", "-i", "./input1.txt", "-e", errorBoundString).Output()
	if err != nil {
		if e, ok := err.(*exec.ExitError); ok {
			fmt.Printf("%s\n", string(e.Stderr))
		}
		log.Fatal(err)
	}

	data := []Segment{}
	json.Unmarshal(out, &data)
	fmt.Println(data)
	segments := Segments{inner: data}
```
for the tables having model file?

Reply:
I also tried to implement this in my PR for you. Take a look. I https://github.com/spongedu/badger/pull/3

Reply:
ok~